examples/edge_bundling.ipynb

```diff
@@ -0,0 +1,524 @@
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client\n",
+    "from pandas import DataFrame\n",
+    "\n",
+    "import fastparquet as fp\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from datashader.bundling import bundle, nop_bundle\n",
+    "from datashader.colors import inferno, viridis\n",
+    "from datashader.utils import export_image\n",
+    "\n",
+    "import datashader as ds\n",
+    "import datashader.transfer_functions as tf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def render_points(df, width=4000, height=4000, cmap=None, bgcolor=None):\n",
+    "    cvs = ds.Canvas(plot_width=width, plot_height=height, x_range=(0, 1), y_range=(0, 1))\n",
+    "    agg = cvs.points(df, 'x', 'y',  ds.count())\n",
+    "    img = tf.shade(agg, cmap=cmap) if cmap else tf.shade(agg)\n",
+    "    img = tf.spread(img)\n",
+    "    return tf.set_background(img, bgcolor) if bgcolor else img"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def render_lines(df, width=4000, height=4000, cmap=None, bgcolor=None):\n",
+    "    cvs = ds.Canvas(plot_width=width, plot_height=height, x_range=(0, 1), y_range=(0, 1))\n",
+    "    agg = cvs.line(df, 'x', 'y',  ds.count())\n",
+    "    img = tf.shade(agg, cmap=cmap) if cmap else tf.shade(agg)\n",
+    "    return tf.set_background(img, bgcolor) if bgcolor else img"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "client = Client()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Edge bundling with graph read from Parquet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "nodes_df = fp.ParquetFile('nodes.snappy.parq').to_pandas(index='id')\n",
+    "edges_df = fp.ParquetFile('edges.snappy.parq').to_pandas(index='id')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "nodes_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "edges_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Graph with only nodes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "render_points(nodes_df, width=2000, height=2000)"
+   ]
```

Comment:
@jbednar, the same happened to me, but when I reran the cell the points appeared.